q2lint.py

```diff
@@ -35,11 +35,18 @@ def main():
                           pathlib.Path('.').glob('**/*/package.json'))
     if npm_packages:
         import subprocess
+        import os
 
         for package in npm_packages:
             pkg_path = package.parent
-            cmd = 'cd %s && npm i && npm run build -- --bail && (git diff ' \
-                  '--quiet */bundle.js || bash -c "exit 42")' % pkg_path
+            install = 'cd %s && npm i --silent' % pkg_path
+            cmd = subprocess.run(install, shell=True,
+                                 stdout=open(os.devnull, 'wb'))
```

Comment:
That's cool.  I tend to rely too heavily on SO, and should spend more time on the actual documentation.  Outsourcing the opening and writing to Subprocess definitely seems advantageous to me.  Thanks for pointing this out!